frontend/public/style/_overrides.scss

```diff
@@ -1,6 +1,16 @@
 // Use this file to override styles from 3rd party dependencies
 $pf-4-nav-bar-height: 76px; // Height of the PatternFly 4 masthead
 
+// Set max height and enable vertical scroll
+.co-alert--scrollable {
```

Comment:
`.co-alert` doesn't exist, so you wouldn't have a modifier.  Perhaps just `.co-alert`?

Reply:
I think I'd put this rule in _common.scss since we're not really overriding existing declartions.